option optimize_for = LITE_RUNTIME;

message LegionChangeLogoAnswer
{
 required uint32 result = 1;
 required uint32 logo_id = 2;
 required uint32 frame_id = 3;
}
